Source/WebCore/platform/graphics/FontGenericFamilies.cpp

```diff
@@ -46,8 +46,7 @@ static bool setGenericFontFamilyForScript(ScriptFontFamilyMap& fontMap, const St
 
 static inline bool computeUserPrefersSimplified()
 {
-    const Vector<String>& preferredLanguages = userPreferredLanguages();
-    for (auto& language : preferredLanguages) {
+    for (auto& language : userPreferredLanguages(ShouldMinimizeLanguages::No)) {
```

Comment:
Ditto.

Reply:
(and ditto from above, reset this and include it in the filed bug)

Reply:
Will do that.